subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -85,6 +85,8 @@
  * test {
  *   // enable TestNG support (default is JUnit)
  *   useTestNG()
+ *   // enable JUnit Platform and JUnit Jupiter Engine (a.k.a. JUnit 5) support
+ *   // useJUnitPlatform()
```

Comment:
This method call should probably not be commented out.

Reply:
Done.